DigitalLearningSolutions.Data/Services/SupervisorService.cs

```diff
@@ -47,7 +47,7 @@ public interface ISupervisorService
         int InsertCandidateAssessmentSupervisor(int delegateId, int supervisorDelegateId, int selfAssessmentId, int? selfAssessmentSupervisorRoleId);
         bool InsertSelfAssessmentResultSupervisorVerification(int candidateAssessmentSupervisorId, int resultId);
         //DELETE DATA
-        bool RemoveCandidateAssessmentSupervisor(int selfAssessmentId, int supervisorDelegateId);

```

Comment:
Removing the selfAssessmentId will result in the supervisor being removed from all self assessments that they are supervising for this delegate rather than just the one the delegate has removed them from.

An ID PK column has been added to CandidateAssessments table since this work was originally done so we could pass just a candidateAssessmentId parameter safely but we can't use only selfAssessmentId.